pages/linux/apt.md

```diff
@@ -4,28 +4,24 @@
 
 - Update list of packages and versions available. This should be run before running further apt commands:
 
-`apt update`
+`sudo apt update`
```

Comment:
Could you explain why its important to add `sudo` to this command.  Does it not function in user mode?  My gut tells me we shouldn't use sudo - and if you are using apt-get you should know to sudo it.  But maybe the other mods feel differently.